app/controllers/application_controller.rb

```diff
@@ -27,4 +29,14 @@ def error(status_code, exception = nil)
   def set_expiry(age = 60.minutes)
     expires_in age, public: true unless Rails.env.development?
   end
+
+  def set_cors_headers
+    headers['Access-Control-Allow-Origin'] = '*'
+    headers['Access-Control-Allow-Methods'] = 'GET'
+    headers['Access-Control-Allow-Headers'] = 'Origin, X-Requested-With, Content-Type, Accept'
```

Comment:
@alexmuller that would be on the response, not the request - I don't think `Content-Type` would be set on a `GET` request.